dev/packages/example/reference/1.0.0/manifest.yml

```diff
@@ -28,6 +28,10 @@ license: basic
 # The default type is integration and will be set if empty.
 type: integration
 
+# Details about the owner of the package. Current only github link is supported.
+# A package can only have a single owner, the owner can be a user or a team.
```

Comment:
@mtojek I added a comment here to make it more obvious it can be a user or a team.

Reply:
Thanks for the explanation. I assume that team aliases will be set for all Elastic packages?

Reply:
Yes, I hope so. This was the initial driver for this in https://github.com/elastic/package-registry/issues/451 Not sure if there is a good way to add this to the generator as in Filebeat for example, some belong to SIEM, some to Platforms team.